scss/base.scss

```diff
@@ -19,3 +19,32 @@
 body {
   font-family: 'NotoSans';
 }
+
+h1 {
+  font-family: DINPro;
+  font-style: normal;
```

Comment:
nit: i dont think you need to add "font-style: normal" since its the default value. same applies to h2 and h3